frontend/src/components/LogDetail/index.tsx

```diff
@@ -137,6 +137,7 @@ function LogDetail({
 	// Go to logs explorer page with the log data
 	const handleOpenInExplorer = (): void => {
```

Comment:
Open in Explorer button is only shown when the log details page is opened in Infra Monitoring page. So other places shouldn't be affected